components/eam/src/control/cam_history.F90

```diff
@@ -2093,7 +2093,7 @@ subroutine AvgflagToString(avgflag, time_op)
     case ('B')
       time_op(:) = 'mean00z'
     case ('I')
-      time_op(:) = ' '
+      time_op(:) = 'point'
```

Comment:
More commonly such fields would be known as "instantaneous". I am not aware if there is a discussion that we should use "point" instead.

Reply:
For the cell_method:time, CF convention says its "point".

Reply:
@rljacob's answer is supported by the list of `cell_methods` values in CF's [Appendix E: Cell Methods](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#appendix-cell-methods).

Reply:
Thanks @rljacob , @klindsay28. I was wondering there might be a reference for that use, but I  didn't find it.